README.md

```diff
@@ -86,21 +86,24 @@ An easy way to try SmartDataCenter is by downloading a Cloud on a Laptop
 (CoaL) build. This is a VMware virtual appliance providing a
 full SDC headnode for development and testing.
 
-Minimum requirements: practically speaking, a good CoaL experience
-requires a **Mac** with at least **16GB** RAM and **SSD** drives. Currently, all
-core team members using CoaL are on Macs with VMware Fusion. For Linux and Windows
-**VMware Workstation should work**, but has not recently been tested.
+The minimum requirements, practically speaking, for a good CoaL experience
+is a **Mac with at least 16 GB RAM and an SSD**. Currently, almost all team
+members using CoaL are on Macs with VMware Fusion. Vmware Workstation for 
+Linux is used by a few in the community. VMware Workstation for Windows
+should work, but has not recently been tested.
 
-See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthrough.
+See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough
+walkthrough including enabling provisioning on the headnode for development.
 
-1. Start the download of the latest CoaL build. The tarball is over 2GB.
+1. Start the download of the latest CoaL build. The tarball is
+   approximately 2 GB.
 
     ```bash
     curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/coal-latest.tgz
     ```
 
 2. Install VMware, if you haven't already.
-    - Mac: [VMware Fusion](http://www.vmware.com/products/fusion) 4, 5 or 7.
+    - Mac: [VMware Fusion](http://www.vmware.com/products/fusion) 5 or 7.
```

Comment:
No, the images are still built in the format that corresponds to VMware Fusion 4.

Do we have people still using Fusion 4? That combined with it's end of availability being 2012/08/23, I thought it made sense to reduce the surface area. If someone comes along and wants to set up with Fusion 4 then we could revisit.

I've also spent a little bit of time trying to improving the script, https://github.com/lloydde/sdc/commit/2af6b8484b513c1092ee79b9befcf884644be239 . I plan to revisit after I help get some of the lower hanging developer documents fruit done.